Crashlytics/Crashlytics/Controllers/FIRCLSReportManager.m

```diff
@@ -12,6 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//
+// The report manager has the ability to send to two different endpoints.
+//
+// The old legacy flow for a report goes through the following states/folders:
+// 1. active - .clsrecords optimized for crash time persistance
+// 2. processing - .clsrecords with attempted symbolication
+// 3. prepared-legacy - .multipartmime of compressed .clsrecords
+//
+// The new legacy flow for a report goes through the following states/folders:
```

Comment:
drive-by comment: remove this extra `legacy`

Reply:
![giphy](https://user-images.githubusercontent.com/555046/74569892-79081080-4f49-11ea-8264-947c123c4d63.gif)